rasa/core/lock_store.py

```diff
@@ -123,9 +123,14 @@ async def _acquire_lock(
                 logger.debug(f"Acquired lock for conversation '{conversation_id}'.")
                 return lock
 
+            items_before_this = ticket - (lock.now_serving or 0)
+            title = "items" if items_before_this > 1 else "item"
```

Comment:
We could use https://pypi.org/project/inflect/ ? 
I think I suggested it once before and was told it was overkill for one time, but have seen this kind of thing a lot in the code base since.

Reply:
What's your experience regarding the speed of it? Loading the `engine` everytime we log this debug message seems quite expensive. I agree with you in general but for `debug` log messages I'm not sure if it's the right thing. I'm even inclined to do something like `because {items_before_this} other item(s) for ....`

Reply:
@joejuzl What do you think?